conf/database_admin.env

```diff
@@ -4,6 +4,9 @@ DB_PASSWD=passwd
 # Optionally set migration version to migrate to
 #SCHEMA_MIGRATION=40
 
+# Optionally set to "true" to reset database schema
+# RESET_SCHEMA=true
```

Comment:
Why do we need this ?

Reply:
It allows to reset db schema to any required version. For instance now we need to downgrade db schema in stage env. We can do it using this without special request to sre.